### ISSUE TITLE
[SITE] Scan the delivered shipment and mark it as Received if the 

### ISSUE THREAD

**Issue by paiusCatalin** [PharmaLedger-IMI/csc-workspace-main#230]
After the shipment is Delivered, SITE should Scan the shipment and validate the shipment. If the shipment is ok, SITE should Sign and mark the shipment as Received

**Comment by paiusCatalin** [comment]
![SITE_Receive_Shipment-11](https://user-images.githubusercontent.com/17168446/132863705-3d9df204-4dd5-411f-be64-f0926b94d9d8.jpg)
![SITE_Receive_Shipment-12](https://user-images.githubusercontent.com/17168446/132863709-0f7c60b8-c481-453c-ad7a-af97deafb37a.jpg)
![SITE_Receive_Shipment-13](https://user-images.githubusercontent.com/17168446/132863712-094447c7-9a3b-48e4-9601-c4c88d56e9e0.jpg)
![SITE_Receive_Shipment-14](https://user-images.githubusercontent.com/17168446/132863713-2e2699a0-b416-4fe4-95df-fff1eaad8d4f.jpg)
![SITE_Receive_Shipment-15](https://user-images.githubusercontent.com/17168446/132863719-c65341a9-3cdb-4786-b5bf-5122a91e7bcc.jpg)
![SITE_Shipment_Received-16](https://user-images.githubusercontent.com/17168446/132863720-a1e3d7d9-2d35-41be-84c4-c999f62110b9.jpg)

**Comment by arijitnv** [comment]
I am currently working on this request.

**Comment by Mastaleru** [comment]
In SITE, in Scan Shipment Controller use the viewModelResolver in order to get the shipment form view-model for the form model..

**Comment by Mastaleru** [comment]
@Mastaleru add the Kits

**Comment by arijitnv** [comment]
Do I need to scan the kits along with shipment?

**Comment by Mastaleru** [comment]
no, just add the kits list from kitIdDSU. I will add them myself
